test/webflow_test.rb

```diff
@@ -129,7 +146,7 @@ def test_it_lists_and_deletes_items
       items = client.items(COLLECTION_ID)
       items.each do |item|
         result = client.delete_item(item)
-        assert_equal({"deleted"=>1}, result)
+        assert_equal({"deleted"=>{}}, result)
```

Comment:
This is strange. We also have this test when testing our `after_destroy` callback: https://github.com/vfonic/webflow_sync/blob/master/spec/jobs/webflow_sync/destroy_item_job_spec.rb#L38